### PR TITLE
Writing argument description to config string (issue #65)

### PR DESCRIPTION
**Pull request by SkyToGround**

### CODE REVIEW

File: include/CLI/App.hpp [hunk 1]
```diff
@@ -809,35 +809,46 @@ class App {
...
+                    value = std::to_string(opt->count());
 
                     // Flag, not present
                 } else if(opt->count() == 0 && default_also) {
-                    out << name << "=false" << std::endl;
+                    value = "false";
+                }
+                
+                if (value.size() != 0) {
+                    if (write_description and opt->description_.size() != 0) {
```
Comment (henryiii) [inline include/CLI/App.hpp]: `and` is not valid here, should be `&&` (Only msvc cares)

Comment (SkyToGround) [reply include/CLI/App.hpp]: This has now been changed.

**Comment by henryiii** [comment]
When you remove the WIP I'll merge, LGTM.

**Comment by SkyToGround** [comment]
Ah, sorry. Forgot about that as I have been taking time off the past week.

**Pull request merged**